docusaurus/docs/operate/configs/relayminer_config.md

```diff
@@ -282,6 +303,41 @@ a service provided by the `Supplier` and served by the `RelayMiner` instance.
 It MUST match the `Service.Id` specified by the supplier when staking for the
 service.
 
+### `signing_key_names`
+
+_`Required`_ if `default_signing_key_names` is empty.
+
+This option specifies the list of signing key names specific to a supplier. If a supplier does not provide its own `signing_key_names`, the `RelayMiner` will use the `default_signing_key_names`.
+
+#### Example Configuration
+
+```yaml
+default_signing_key_names:
+  - supplier_key2
+  - supplier_key3
+suppliers:
+  # This supplier will only run for `supplier_key1` and `supplier_key4` keys
+  - service_id: supplier1
```

Comment:
I'm a bit confused:

1. Can you avoid using `supplier` for `service_id`?
2. Do you mean `This supplier will ...` or `This relayminder will ...`

In either case, can you try to put yourself in the shoes of someone who has never thought about this and make it really obvious how this works? Even I'm confused...

Maybe add a mermaid diagram if its applicable.

Reply:
1. 👍
2. Reworded as "RelayMiner will only provide service for `llm-model` using `supplier_key1` and `supplier_key4` keys."